src/Components/common/getUrlInputFile.js

```diff
@@ -0,0 +1,12 @@
+
+export const UrlInput = (e,setUrl,setFieldValue) =>{
+
+  const imageFile = e.target.files[0];
```

Comment:
this imageFile should come from the parameters because its specific of the TestimonialForm